fast-af/fast-checkout-woocommerce
# PINT-985 - Add button wrapper content option

# Description

Add an option to be able to add content above or below the Fast Checkout buttons.

# Testing instructions

1. Login to the staging admin.
2. Go to the Options tab in the Fast settings page.
3. Add content to "Extra Button Content"
4. Select "Before" or "After" for "Location of Extra Button Content"
5. Save the changes to the settings.
6. Verify that the content you added is displayed before or after the checkout buttons

# Checks
